src/Ledger/Deleg.lagda

```diff
@@ -94,71 +108,63 @@ private variable
   poolParams : PoolParams
   wdrls  : RwdAddr ⇀ Coin
 
-module _ (open PParams) where
 \end{code}
 \begin{code}
-  cwitness : DCert → Credential
-  cwitness (delegate c _ _ _)  = c
-  cwitness (dereg c)           = c
-  cwitness (regpool c _)       = c
-  cwitness (retirepool c _)    = c
-  cwitness (regdrep c _ _)     = c
-  cwitness (deregdrep c)       = c
-  cwitness (ccreghot c _)      = c
-
-  getDRepVote : GovVote → Maybe Credential
-  getDRepVote record { role = DRep ; credential = credential }  = just credential
-  getDRepVote _                                                 = nothing
+getDRepVote : GovVote → Maybe Credential
+getDRepVote record { role = DRep ; credential = credential }  = just credential
+getDRepVote _                                                 = nothing
 \end{code}
 \caption{Functions used for CERTS transition system}
 \end{figure*}
 
 \begin{figure*}[h]
 \begin{code}
 data _⊢_⇀⦇_,DELEG⦈_ : DelegEnv → DState → DCert → DState → Set where
-  DELEG-delegate :
-    (c ∉ dom rwds → d ≡ pp .PParams.poolDeposit)
-    → (c ∈ dom rwds → d ≡ 0)
-    → mc ∈ mapˢ just (dom pools)
-    ────────────────────────────────
-    ⟦ pp , pools ⟧ᵈᵉ ⊢  ⟦ vDelegs , sDelegs , rwds ⟧ᵈ ⇀⦇ delegate c mv mc d ,DELEG⦈
-                        ⟦ insertIfJust c mv vDelegs , insertIfJust c mc sDelegs , rwds ∪ˡ ❴ c , 0 ❵ ⟧ᵈ
+  DELEG-delegate : let open PParams pp in
```

Comment:
The previous version seems a bit nicer to my eyes, since it avoids the use of non-standard notation.  (I had to look up the bullet point syntax to make sure it doesn't what I suspected.)  I can see the appeal of the bullet points for readers of the pdf though.

Alternatively, I like the style of moving the arrows to the far left---see example below---which I got used to when adopting Martin Escardo's Type Topology library style in the `agda-algebras` library; it gives the appearance a list of antecedents above the horizontal line without requiring non-standard syntax:

```agda
  DELEG-delegate : let open PParams pp in
          (c ∉ dom rwds → d ≡ poolDeposit)
    →     (c ∈ dom rwds → d ≡ 0)
    →     mc ∈ mapˢ just (dom pools)
          ────────────────────────────────
          ⟦ pp , pools ⟧ᵈᵉ ⊢
            ⟦ vDelegs , sDelegs , rwds ⟧ᵈ ⇀⦇ delegate c mv mc d ,DELEG⦈
            ⟦ insertIfJust c mv vDelegs , insertIfJust c mc sDelegs , rwds ∪ˡ ❴ c , 0 ❵ ⟧ᵈ
```

Reply:
This reformatting is in line with #340. I'm ok with changing the style, but discussion should happen there.